sys/include/ztimer.h

```diff
@@ -297,6 +297,9 @@ struct ztimer_clock {
     uint32_t lower_last;            /**< timer value at last now() call     */
     ztimer_now_t checkpoint;        /**< cumulated time at last now() call  */
 #endif
+#if MODULE_PM_LAYERED
+    int8_t required_pm_mode;        /**< min. pm mode required for the clock to run */
```

Comment:
Not sure about using `-1` here as a value. `0` is idle and then we have four additional modes. I would be a fan of `255` as an invalid mode, as it is also "below" the deepest sleep mode.

Reply:
Okay good point.

Thinking about this a second time, I would rather define  `ZTIMER_NO_REQUIRED_PM_MODE` and use this instead of checking magic numbers. Would you agree?

Reply:
+1